gemrb/core/Plugin.h

```diff
@@ -84,16 +84,31 @@ class GEM_EXPORT ImporterBase : public Held<ImporterBase>
 	}
 };
 
+/* MSVC must not see an abstract type _anywhere_ going into a `new` or `make_shared`. */
+template<class T, typename... ARGS>
```

Comment:
how are these different from `std::make_shared`?

Reply:
They are not. It just stops MSVC from seeing abstract `DialogMgr` & co types on a potential instantiation path.

Reply:
it sounds like you are saying they are _not_ different.

Reply:
we should only be using _any_ make shared (std or our own) on concrete types. The non concrete users should use `std::static_ptr_cast`

Reply:
I can rename that thing into `MakeImporter` or so since I, for now, only see this as a the only working way (with MS) to compile `ImporterPlugin` with either type appearance, so there is nothing we'd wanna be doing with non-concrete types here anyway. If I wanted to generally use it across the codebase, sure.

Reply:
so this is only used on the line below then? yeah, lets rename it to not conflate it.